_posts/2015-04-06-Simplified-Workspace-Creation.md

```diff
@@ -29,7 +29,7 @@ update your `~/.bazelrc` file.
 finds/generates, you can create a `tools/` directory in your project and
 Bazel will attempt to use that instead of the system-wide one.
 
-See the [getting started]({{ site_root }}/docs/getting-started.html) docs for more info about
+See the [getting started](site.docs_site_url/getting-started.html) docs for more info about
```

Comment:
The variable needs to be surrounded with braces: `{{ site.docs_site_url }}`

Reply:
done in both pull requests.